qiita_pet/support_files/doc/source/faq.rst

```diff
@@ -90,10 +90,26 @@ A few more instructions: for the example above the workflow should be:
 
 .. _issues_unzip:
 
-How to solve unzip errors?
---------------------------
-
-When downloading large zip files within Qiita there is a change that you will get
+How to solve download or unzip errors?
+--------------------------------------
+
+Dealing with large files might be daunting but, in general, following these
+instructions should make things easier. First, make sure that you have enough space
+for the zip download file; for the American Gut, you should have at least close to 700Gb
+but having close to a Tera (1000Gb) should be better as you will need to uncompress
+the zip file. Second, make sure that your computer has all the sleep settings turned off;
+for example, in a Mac, got to System Preferences, Energy Saver, Power Adapter and unselect
+the option of "Put hard disks to sleep when possible"; don't forget to save the settings.
+Third, download the file but point to the storage that you want to save your file in; using
+Chrome, right click on the download button and select "Save Link As ..."; and select the
+location where you have enough space (see point 1). Fourth, wait for the download to finish,
+around 13hrs in an OK USA Internet connection for the AGP but this will depend on your Internet
+service. Finally, unzip de file with a newer version of zip.
```

Comment:
de -> the

What is a newer version of zip?

Reply:
Oh, I see below. Might want to mention "(see below)". Note that this is likely limited to versions of OSX. unzip 6.0.0 on 10.13 reports a compilation date of 2017, and the unzip 6.0.0 build on barnacle reports a compilation date of 2010

Reply:
ha! de! thanks for the info ...